packages/component-library/stories/ScreenGridMap.notes.md

```diff
@@ -0,0 +1,25 @@
+# ScreenGrid Map Component
+
+## Standard
+
+The Standard story shows the standard usage of the ScreenGrid Map Component for the CIVIC platform. ScreenGrid Map includes the standard styling.
+
+These properties can be set in the standard story:
+
+- `colorRange`
+- `cellSizePixels`
+- `opacity`
+- `data`
+- `getPosition`
+
+## Custom
+
+The Custom story shows all the possible properties that can be passed to the ScatterPlot Map Component for customization.
```

Comment:
ScatterPlot  should read ScreenGrid

Reply:
Good eyes! I fixed this in my latest commit.